docs/reference/dapr-dashboard.md

```diff
@@ -0,0 +1,19 @@
+# dapr dashboard
+
+## Description
+
+Starts Dapr dashboard in a Kubernetes cluster.
```

Comment:
`dapr dashboard ` is for standalone and kubernetes.  Only when invoked with flag -k , it starts for kubernetes.
So you might need to change "Start Dapr dashboard" for now. Since with current changes, it is not supported for standalone.

Reply:
Since the dashboard deployment has to be done before running dashboard command. 
It would be better to state that in the doc and command help about this pre-reqs:

Prerequisites : Dapr dashboard should be deployed in the Kubernetes cluster. 
You can deploy the dashboard in the Kubernetes cluster as follows:
kubectl apply -f https://raw.githubusercontent.com/dapr/dashboard/master/deploy/dashboard.yaml "

Reply:
Gotcha! I'll go ahead and update the docs. For dashboard deployment, I remember Yaron mentioning that `dapr init` would take care of it. Is that issue going to be tackled in this milestone?

Reply:
Yes the plan is  that `dapr init` will take care of,  yes it is targeted to this milestone but might come later. Due to that, till it comes, it is better to add the prereqs in the doc, later it can be removed from doc once `dapr init` takes care of.